lib/desktop/cli.rb

```diff
@@ -30,6 +30,7 @@
 require 'tty/reader'
 require 'commander'
 require 'erb'
+require '/code/flight-desktop/lib/desktop/patches/geometry_option.rb'
```

Comment:
This code path is not going to work for the RPM build.  Well it won't work when installed on a customer machine as they are unlikely to have anything at `/code/flight-desktop`.

When requiring files that are in the project itself, its quite common to use `require_relative` and a relative path.

`require_relative` looks for a path relative to the current working directory of the process.  For Flight Desktop it is safe to assume that the working directory will be root of the repo.

So requiring the geometry option becomes: `require_relative "patches/geometry_option.rb"`.